src/main/webapp/dev/js/main/studentMotd.es6

```diff
@@ -6,7 +6,8 @@ function fetchMotd(motdUrl, motdContentSelector, motdContainerSelector) {
         type: 'GET',
         url: `${window.location.origin}/${motdUrl}`,
         success(data) {
-            $(motdContentSelector).html(data);
+            const bodyContent = data.match(/<body[^>]*>[\s\S]*<\/body>/gi);
```

Comment:
It turns out that `.` matches everything **except line breaks**. That's why it doesn't work. So I guess we can just go with `[\s\S]`.